lib/ecommerce/attribute_handler.rb

```diff
@@ -0,0 +1,51 @@
+require 'bigdecimal'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.